### PR TITLE
feat(a380x): OANS advanced guide: Add fallback mode documentation

### PR DESCRIPTION
**Pull request by flogross89**
## Summary

Adds docs for the Navigraph-less fallback mode, after feedback from the last days. 

### Location

Discord username (if different from GitHub):

### CODE REVIEW

File: docs/pilots-corner/a380x/a380x-advanced-guides/oans-btv.md [hunk 1]
```diff
@@ -68,6 +70,19 @@ The upper left of the ND now shows:
 ![No selection](../assets/advanced-guides/oans-btv/btv-setup-no-selection.png)
 ![Runway + exit selected](../assets/advanced-guides/oans-btv/btv-setup-selected.png)
 
+To finalize the setup, the auto brake mode selector knob is turned to the "BTV" option, which arms BTV (this should be
```
Comment (Valastiri) [inline docs/pilots-corner/a380x/a380x-advanced-guides/oans-btv.md]: The flow for the image here seems a bit weird. I have a couple of suggestions.

1. Add a header titled `Finalize BTV Setup` in an H3. And link to that to separate it as finalizing BTV IMO is concurrent with both having navigraph or not.
2. Wrap the without navigraph unlimited in a `???` admonition to collapse it and users can open that at will if they don't have it.

Suggestions will be provided below via review.

File: docs/pilots-corner/a380x/a380x-advanced-guides/oans-btv.md [hunk 2]
```diff
@@ -68,6 +70,19 @@ The upper left of the ND now shows:
...
+### Without Navigraph Unlimited
+
+Provided the landing runway is set in the FMS, BTV can also be set up without the airport map from the OANS. Turn the EFIS range
+selector to ZOOM, and open the OANS control panel by clicking on the arrow symbol on the lower right of the ND.
+
+Change to the "MAP DATA" tab, check that the selected landing runway is correct and enter the desired stopping distance in the input
+field next to "BTV STOP DISTANCE" (don't forget to finish by pressing the enter key).
+
+![BTV STOP DISTANCE selection in OANS control panel](../assets/advanced-guides/oans-btv/btv-fallback.png)
+
```
Comment (Valastiri) [inline docs/pilots-corner/a380x/a380x-advanced-guides/oans-btv.md:85]:
```suggestion
### Finalize BTV Setup
```

In line with my option 1 above.

File: docs/pilots-corner/a380x/a380x-advanced-guides/oans-btv.md [hunk 3]
```diff
@@ -68,6 +70,19 @@ The upper left of the ND now shows:
 ![No selection](../assets/advanced-guides/oans-btv/btv-setup-no-selection.png)
 ![Runway + exit selected](../assets/advanced-guides/oans-btv/btv-setup-selected.png)
 
+To finalize the setup, the auto brake mode selector knob is turned to the "BTV" option, which arms BTV (this should be
+checked in the PFD FMA). Images [see below](#without-navigraph-unlimited).
```
Comment (Valastiri) [inline docs/pilots-corner/a380x/a380x-advanced-guides/oans-btv.md]:
```suggestion
checked in the PFD FMA). Images [see below](#finalize-btv-setup).
```
in line with option 1 above

File: docs/pilots-corner/a380x/a380x-advanced-guides/oans-btv.md [hunk 4]
```diff
@@ -68,6 +70,19 @@ The upper left of the ND now shows:
...
+
+### Without Navigraph Unlimited
+
+Provided the landing runway is set in the FMS, BTV can also be set up without the airport map from the OANS. Turn the EFIS range
+selector to ZOOM, and open the OANS control panel by clicking on the arrow symbol on the lower right of the ND.
+
+Change to the "MAP DATA" tab, check that the selected landing runway is correct and enter the desired stopping distance in the input
+field next to "BTV STOP DISTANCE" (don't forget to finish by pressing the enter key).
+
+![BTV STOP DISTANCE selection in OANS control panel](../assets/advanced-guides/oans-btv/btv-fallback.png)
```
Comment (Valastiri) [inline docs/pilots-corner/a380x/a380x-advanced-guides/oans-btv.md]:
```suggestion
??? info "Without Navigraph Unlimited"
	### Without Navigraph Unlimited
	
	Provided the landing runway is set in the FMS, BTV can also be set up without the airport map from the OANS. Turn the EFIS range
	selector to ZOOM, and open the OANS control panel by clicking on the arrow symbol on the lower right of the ND.
	
	Change to the "MAP DATA" tab, check that the selected landing runway is correct and enter the desired stopping distance in the input
	field next to "BTV STOP DISTANCE" (don't forget to finish by pressing the enter key).
	
	![BTV STOP DISTANCE selection in OANS control panel](../assets/advanced-guides/oans-btv/btv-fallback.png)
```

In line with option 2.

Must keep h3 inside to have it in the TOC and for the rest of your internal links.

**Review by Valastiri — CHANGES_REQUESTED** [review]
Minor flow considerations

**Comment by flogross89** [comment]
@Valastiri addressed your comments now, and furthermore removed one duplicate paragraph about finalizing the setup.

**Pull request merged**